jobs/apply-specs/templates/specs/kubernetes-dashboard/kubernetes-dashboard.yml.erb

```diff
@@ -114,7 +96,9 @@ spec:
         - containerPort: 8443
           protocol: TCP
         args:
+        <%- if !p('delete-heapster') then -%>
           - --heapster-host=https://heapster.kube-system.svc.cluster.local:8443
+        <%- end -%>
```

Comment:
`heapster-ca` VolumeMount and volume should be encapsulated between `<%- if !p('delete-heapster') then -%>` and `<%- end -%>`